.github/workflows/Chibi_ci_build.yml

```diff
@@ -0,0 +1,72 @@
+# preforms CI builds using compose.py from
+# https://github.com/CleverRaven/Cataclysm-DDA
+#
+# This action is triggerd by any PR against the master branch as well
+# as on any push to the master branch itself
+# that changes any file in gfx/UltimateCataclysm, gfx/MShockXotto+/ or gfx/Chibi_Ultica/ folders
+name: Chibi_Ultica composer
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'gfx/UltimateCataclysm/**'
+      - 'gfx/MShockXotto\+/pngs_large_64x80/monsters/**'
+      - 'gfx/MShockXotto\+/pngs_tiles_32x32/character/**'
+      - 'gfx/MShockXotto\+/pngs_tiles_32x32/monster/**'
+      - 'gfx/MShockXotto\+/pngs_tall_32x48/**'
+      - 'gfx/Chibi_Ultica/**'
+      - '.github/workflows/Chibi_ci_build.yml'
+  pull_request:
+    branches:
+      - master
+    paths:
+      - 'gfx/UltimateCataclysm/**'
```

Comment:
Will this action take precedence over Ultica builder?

Reply:
I was assuming that both action would take place, but I don't know.